package.json

```diff
@@ -41,10 +41,16 @@
   "bugs": {
     "url": "https://github.com/react-native-kit/react-native-track-player/issues"
   },
-  "scripts": {
-  },
+  "scripts": {},
   "peerDependencies": {
     "react": ">=16.8.6",
     "react-native": ">=0.60.0-rc.2"
+  },
+  "dependencies": {
+    "react": "16.13.1",
+    "react-native": "0.63.2"
+  },
```

Comment:
A community module shouldn't declare these dependencies directly, These should be devDependencies instead.

Reply:
Thanks!